code/game/objects/structures/ladders.dm

```diff
@@ -197,3 +197,61 @@
 
 /obj/structure/ladder/crafted
 	crafted = TRUE
+
+/obj/structure/ladder/unbreakable/teleport
+	name = "mysterious ladder"
+	desc = "A mysterious ladder leading down. Where does it go?"
+
+/obj/structure/ladder/unbreakable/teleport/Initialize(mapload, obj/structure/ladder/up, obj/structure/ladder/down)
+	AddElement(/datum/element/undertile, TRAIT_T_RAY_VISIBLE, use_anchor = TRUE)
+	return ..()
+
+/obj/structure/ladder/unbreakable/teleport/LateInitialize()
```

Comment:
this won't be called unless Initialize returns INITIALIZE_HINT_LATELOAD

Reply:
The parent ladder does that (Line 25)

Reply:
🙏